docs/en/ufs/Ozone.md

```diff
@@ -84,6 +84,65 @@ $ ./bin/alluxio fs mount \
   /ozone o3fs://<OZONE_BUCKET>.<OZONE_VOLUME>/
 ```
 
+Possible core-site.xml and ozone-site.xml
+- core-site.xml
```

Comment:
can you put all the references to `core-site.xml` and `ozone-site.xml` inside of backticks ("`")? IMO it makes it easier to read and stand out more.

Reply:
Done